### PR TITLE
feat: require pullSecret only for enterprise edition

### PR DESCRIPTION
**Pull request by j-zimnowoda**

### CODE REVIEW

File: values/k8s/k8s-raw.gotmpl [hunk 1]
```diff
@@ -74,6 +74,7 @@ resources:
     - kind: ServiceAccount
       name: default
       namespace: team-admin
+  {{- if eq $v.otomi.mode "ee" }}
```
Comment (Morriz) [inline values/k8s/k8s-raw.gotmpl:77]: I am trying to keep that flag out, as we should just check for the existence of the pullsecret. It was not quickly possible in our setup to nullify it

**Review by Morriz — CHANGES_REQUESTED** [review]

File: values/k8s/k8s-raw.gotmpl (hunk 1 above)
Comment (j-zimnowoda) [reply values/k8s/k8s-raw.gotmpl:77]: I do not understand your concept. What would you like to achieve? And does it gonna help other users/developers?

Comment (Morriz) [reply values/k8s/k8s-raw.gotmpl:77]: don't use that flag, it will be removed

Comment (j-zimnowoda) [reply values/k8s/k8s-raw.gotmpl:77]: ok, I will change the code but next time please mark this field as deprecated and communicate it to the team.

Comment (j-zimnowoda) [reply values/k8s/k8s-raw.gotmpl]: done

**Review by Morriz — APPROVED** [review]

**Pull request merged**